packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -170,6 +170,11 @@ export default class SearchBox extends Base {
 
   render() {
     return html`
+      <style>
+        sfx-search-box {
+          display: block;
```

Comment:
I think search box should be `inline-block`. One should be able to display this component alongside some text. For the record, the computed display styles of an input box and a button are "inline" (Firefox)/"inline-block" (Chrome) and "inline-block" respectively